homeassistant/components/google/calendar.py

```diff
@@ -265,49 +339,41 @@ def _event_filter(self, event: Event) -> bool:
             return True
         return event.transparency == OPAQUE
 
+    async def async_added_to_hass(self) -> None:
+        """When entity is added to hass."""
+        await super().async_added_to_hass()
+        self.coordinator.async_update_listeners()
```

Comment:
This will cause a double call to write state during entity addition. It's better to put the coordinator data parsing in a separate helper method which we can call both here and in `_handle_coordinator_update`.

Reply:
👍🏼